src/App.js

```diff
@@ -1,13 +1,26 @@
 import React from "react";
-
 import Bookings from "./Bookings";
+import TouristInfoCards from "./TouristInfoCards";
+import Footer from "./Footer";
+import Restaurant from "./Restaurant";
+
 import "./App.css";
 
 const App = () => {
   return (
     <div className="App">
-      <header className="App-header">CYF Hotel</header>
+      <heading className="App-header">
```

Comment:
Exercise 2 asks you to "extract" the `<header>` to a component. That means you should move it to a separate component (in a separate file). Do you think you can do that?

Reply:
ok yes. thanks. I did now.